exercises/a-safe-space-for-nulls/problem/problem.md

```diff
@@ -0,0 +1,77 @@
+Create a program that exports a `User` object instance to a CSV file using the null safe operator to access it's member properties.
+
+You will have a variable named `$user` available in your PHP script. We will put it there, each time your program runs  and it will be populated with random data each time it runs.
+
+Sometimes, a number of the properties may not exist and will be null. 
```

Comment:
may not exist kind of sounds like it object won't have the property at all, maybe just `properties won't have a value set and will be null` would be better?

Reply:
Yes loads better 👌